cmd/host_windows.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/rancher-sandbox/rancher-desktop-host-resolver/pkg/commands"
+	"github.com/spf13/cobra"
+)
+
+// hostCmd represents the AF_VSOCK host process that run in a host machine.
+// It receives all the DNS queries from vsock-peer over the AF_VSOCK connection for resolution.
+// Upon startup it attempts to finds the peer process over AF_VSOCK connection by establishing
+// a handshake and verifying a seed string to make sure it is communicating with the right VM.
+var (
+	hostCmd = &cobra.Command{
+		Use:   "vsock-host",
+		Short: "Host-resolver vsock-host process",
+		Long: `Vsock-host runs the host-resolver as a host for AF_VSOCK in a host machine.
+It handles and forwards both TCP and UDP DNS queries over a virtual socket for the peer
+that runs inside a VM.
+
+--------------------HOST-------------------------------------WSL DISTRO------------
+| vsock-host | <----- AF_VSOCK -----> [ VM ] <----- AF_VSOCK -----> | vsock-peer |
+-----------------------------------------------------------------------------------`,
```

Comment:
Oops, didn't notice that your chart isn't lined up anymore…
```suggestion
--------------------HOST------------------------------------WSL DISTRO------------
| vsock-host | <----- AF_VSOCK -----> [ VM ] <----- AF_VSOCK -----> | vsock-peer |
----------------------------------------------------------------------------------`,
```